Add Categories section in CONTRIBUTING.md (#7393)
jdavis/twofactorauth: CONTRIBUTING.md

| developer        |         Developer         | Development software                                                             |
| domains          |          Domains          | DNS Registrars                                                                   |
| education        |         Education         | Non-university education platforms                                               |
| email            |           Email           | Email providers                                                                  |
| entertainment    |       Entertainment       | Audio/Video entertainment excluding games                                        |
| finance          |          Finance          | Financial, insurance and pension services                                        |
| food             |           Food            | Food and beverage services                                                       |
| gaming           |          Gaming           | Games and game platforms. Sites for buying games should be listed in Retail      |
| government       |        Government         | Government portals. Excluding education                                          |

> | health           |          Health           | Health and fittness platforms                                                    |

Reviewer: ```suggestion
| health           |          Health           | Health and fitness platforms                                                     |
```

| iot              |            IoT            | Internet of Things and device management platforms                               |
| legal            |           Legal           | Legal aid services                                                               |
| marketing        |   Marketing & Analytics   | Marketing campaign providers and analytics services                              |
| payments         |         Payments          | Payment providers                                                                |
| post             |     Post and Shipping     | Postal and logistics providers                                                   |
| remote           |       Remote Access       | Remote device access platforms                                                   |
| retail           |          Retail           | Online retail platforms                                                          |
| security         |         Security          | Online security and anti-malware services. Excluding VPN and identity management |
| social           |          Social           | Social networks                                                                  |

> | task             |      Task Management      | Task Management and ToDo-planning services                                       |

Reviewer: ```suggestion
| task             |      Task Management      | Task management and to-do services                                       |
```

| transport        |         Transport         | Transportation services including public transport and airlines                  |
| universities     |       Universities        | University online platforms                                                      |

> | utilities        |         Utilities         | Household utilities including electricity,gas,water,phone and internet providers |

Reviewer: Is the lack of a space after the commas to keep the column width fixed? Because it looks off to me.
